Salary App/Base/Views/SalaryBreakdown/AccordionView.swift

```diff
@@ -0,0 +1,303 @@
+        return scrollView
+    }()
+    
+    
+    private lazy var horizontalStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .leading
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.backgroundColor = .white
+        return stackView
+    }()
+    
+    private lazy var iconButton: UIButton = {
+        let iconButton = UIButton()
+        iconButton.translatesAutoresizingMaskIntoConstraints = false
+        iconButton.setTitle("Closed ▲", for: .normal)
+        iconButton.setTitleColor(.black, for: .normal)
+        iconButton.addTarget(self, action: #selector(touchHappen), for: .touchUpInside)
+        iconButton.tintColor = .black
+        return iconButton
+    }()
+    
+    
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 32
+        return stackView
+    }()
+    
+    private lazy var mainstackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 32
+        stackView.layer.cornerRadius = 16
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UIEdgeInsets(top: 16, left: 16, bottom: 48, right: 16)
+        stackView.backgroundColor = .white
+        return stackView
+    }()
+    
+    
+    private lazy var monthlyStackView: SalaryHorizontalStackView = {
+        let stackView = SalaryHorizontalStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+    private lazy var salaryMonthlyLbl: UILabel = {
+        let salaryMonthlyLbl = UILabel()
+        salaryMonthlyLbl.textColor = .black
+        salaryMonthlyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryMonthlyLbl
+    }()
+    
+    private lazy var yearlyStackView: SalaryHorizontalStackView = {
+        let stackView = SalaryHorizontalStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+    private lazy var salaryYearlyLbl: UILabel = {
+        let salaryYearlyLbl = UILabel()
+        salaryYearlyLbl.textColor = .black
+        salaryYearlyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryYearlyLbl
+    }()
+    
+    private lazy var dailyStackView: SalaryHorizontalStackView = {
+        let stackView = SalaryHorizontalStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
You don't need to set this here all this configuration should be done in the setup function within the component, this is the same for the other views. This should just be 

```

private lazy var dailyStackView: SalaryHorizontalStackView = {
        let vw = SalaryHorizontalStackView()
        return vw
}

```

All styling should be done inside of the component